docs/platform/howto/attach-vpc-aws-tgw.md

```diff
@@ -1,130 +1,180 @@
+   any IP range (CIDR) to be routed using the VPC attachment.
+   :::
 
-The state of the Aiven peering connection is `APPROVED`.
+   The IP ranges will be referred to as `USER_PEER_NETWORK_CIDR`.
+
+## Request the attachment
+
+### Send the connection request
+
+To create the Transit Gateway - VPC attachment, make a request to the Aiven API for a
+peering connection.
 
 :::note
-You can use the `--user-peer-network-cidr` argument multiple
-times to define more than one peer network CIDR. You can also
-create the attachment without any CIDRs and add them later. In this case, the
-attachment will be not be of any use until that is done since no
-addresses will be routed through the TGW from the Project VPC.
+In the request, you can use the `--user-peer-network-cidr` argument multiple times to
+define more than one peer network CIDR. You can also create the attachment without a CIDR,
+but you would need to add it later to make your attachment operational.
 :::
 
-### Accept AWS Transit Gateway VPC attachment
+Depending on the type of your Aiven VPC, select **Project VPC** or **Organization VPC**,
+and run the provided commands, replacing placeholders with meaningful values.
+
+   <Tabs groupId="group1">
+   <TabItem value="pj-vpc" label="Project VPC" default>
+```bash
+avn vpc peering-connection create       \
+  --project-vpc-id VPC_ID               \
+  --peer-cloud-account USER_ACCOUNT_ID  \
+  --peer-vpc USER_TGW_ID                \
+  --user-peer-network-cidr USER_PEER_NETWORK_CIDR
+```
+   </TabItem>
+   <TabItem value="org-vpc" label="Organization VPC">
+```bash
+avn organization vpc peering-connection create \
+  --organization-id ORG_ID                     \
+  --organization-vpc-id VPC_ID                 \
+  --peer-cloud-account USER_ACCOUNT_ID         \
+  --peer-vpc USER_TGW_ID                       \
+  --user-peer-network-cidr USER_PEER_NETWORK_CIDR
+```
+   </TabItem>
+   </Tabs>
+
+Once submitted, the request gets the `APPROVED` status, and the Aiven Platform starts
+building the connection by creating the AWS Transit Gateway VPC attachment, which can take
+a few minutes. Once the connection is built, the status changes to `PENDING_PEER`.
+
+### Check the connection status
+
+To check the status of your peering connection request, select either **Project VPC** or
+**Organization VPC**, depending on your Aiven VPC type, and run the provided commands,
+replacing placeholders with meaningful values.
+
+   <Tabs groupId="group1">
+   <TabItem value="pj-vpc" label="Project VPC" default>
+```bash
+avn vpc peering-connection get         \
+  --project-vpc-id VPC_ID              \
+  --peer-cloud-account USER_ACCOUNT_ID \
+  --peer-vpc USER_TGW_ID -v
+```
+   </TabItem>
+   <TabItem value="org-vpc" label="Organization VPC">
+```bash
+avn organization vpc get               \
+  --organization-id ORG_ID             \
+  --organization-vpc-id VPC_ID         \
+  --peer-cloud-account USER_ACCOUNT_ID \
+  --peer-vpc USER_TGW_ID -v
```

Comment:
This command does not have a `-v`flag. `vpc peering-connection-get` seems to have it, though I'm not entirely sure what it does, if anything. But this command does not have such a flag.

It also does not have flags `--peer-cloud-account` and `--peer-vpc`. Those are properties of individual peering connections, not something that identify a VPC.

A complete invocation of the `organization vpc get` command looks like this:
```
avn organization vpc get --organization-id org4f9ed964bca --organization-vpc-id 6a5542cf-2a10-416e-80c2-dcaeddc99a9a
```

Reply:
Apologies, for some reason I missed updating the flags after changing the command. Thanks for catching that!